common/styles/base/_layout.scss

```diff
@@ -37,4 +37,11 @@ $tweakpoints: (
 // Without the width specified below it can cause issues with centering of the images, such as on the work page
 noscript {
   width: 100%;
+  background: color('white');
+  color: color('black');
+  display: inline-block;
```

Comment:
why do we need to do this?

Reply:
The `inline-block`? It shrink-wraps the `noscript` around the image it contains (it's `inline` by default).